gui/qt/history_list.py

```diff
@@ -44,6 +44,20 @@
     "confirmed.png",
 ]
 
+class HistoryTreeWidgetItem(QTreeWidgetItem):
+    def __init__(self, parent=None):
+        QTreeWidgetItem.__init__(self, parent)
+    
+    def __lt__(self, otherItem):
+        column = self.treeWidget().sortColumn()
+        if column == 0:
+            return float( self.data(column, Qt.UserRole+1) ) < float( otherItem.data(column, Qt.UserRole+1) )
```

Comment:
Hey man -- just to be super duper extra special paranoid, two suggestions: 


1. Cast to int if you can.  The reason being as confirmation count gets huge -- say in the 300,000's and beyond, 32 bit floats stop being accurate and start losing TONS of precision.   You can have situations where 2122631231 <= 30000000 and stuff like that.  Floats don't deal well with huge numbers.

2. Even more paranoid; but entirely at your discretion: I'd move this if column==0: float (self.data()) call into the try clause.  Just my own defensive programming habit at play here.  Because data is a QVariant and those things can blow up in your face is someone does something evil such as stuff another data type into Qt.UserRole+1.  Entirely up to you though. Then again.. maybe having it explode as soon as someone introduces a bug and stuffs the wrong data in.. is a "feature". :)

Reply:
Reasonable suggestions, especially since status+conf will always be int. I'm not too worried about float precision, because aren't nearly all python float implementations 64 bits? Anyway, changes made! Thanks!

Reply:
Ha.. I didn't know that they are.  They might be.

I'm old school.  I remember the days when floats were all 32-bits!  Yeah.. if they're 64 bits.. that's pretty good for this app's needs anyway.